api/api.yaml

```diff
@@ -0,0 +1,619 @@
+      parameters:
+        - $ref: '#/components/parameters/State'
+      operationId: VerifierPageReceiveCredential
+      summary: Presents a rendered page, showing the credential associated with the requested state
+      responses:
+        '200':
+          description: A rendered page, showing the credential
+          content:
+            text/html:
+              schema:
+                type: object
+  /verifier/api/v1/accessprotectedservice:
+    get:
+      tags:
+        - verifier
+        - frontend
+      operationId: VerifierPageAccessProtectedService
+      summary: Presents a rendered page, containing the response from the protected service
+      description: When called, the vc connected to the current session is used to access the 'protected-service'. The response then is rendered into an html page.
+      responses:
+        '200':
+          description: A rendered page, showing the response from the protected service
+          content:
+            text/html:
+              schema:
+                type: object
+  /verifier/api/v1/poll/{state}:
+    get:
+      tags:
+        - verifier
+        - api
+      parameters:
+        - $ref: '#/components/parameters/State'
+      operationId: VerifierAPIPoll
+      summary: Endpoint to check the current state of a session
+      responses:
+        '200':
+          description: Returns either the state of the session(pending, expired) or the credential itself
+          content:
+            text/plain:
+              schema:
+                type: string
+  /verifier/api/v1/token/{state}:
+    get:
+      tags:
+        - verifier
+        - api
+      parameters:
+        - $ref: '#/components/parameters/State'
+      operationId: VerifierAPIToken
+      summary: Return the jwt for the requested session
+      responses:
+        '200':
+          description: The access token
+          content:
+            text/plain:
+              schema: 
+                type: string
+        '403':
+          description: Unauthorized, in case the session is not authorized
+  /verifier/api/v1/startsiop:
+    get:
+      tags:
+        - verifier
+        - api 
+      parameters:
+        - $ref: '#/components/parameters/QueryState'
+      operationId: VerifierAPIStartSIOP
+      summary: Initiates the siop flow and returns the 'openid://...' connection string
+      responses:
+        '200': 
+          description: The connection string 
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: openid://?scope=dsba.credentials.presentation.PacketDeliveryService&response_type=vp_token&response_mode=post&client_id=did:key:z6MktZy7CErCqdLvknH6g9YNVpWupuBNBNovsBrj4DFGn4R1&redirect_uri=http://localhost:3000/verifier/api/v1/authenticationresponse&state=&nonce=BfEte4DFdlmdO7a_fBiXTw
+  /verifier/api/v1/authenticationrequest:
+    $ref: '#/paths/~1verifier~1api~1v1~1startsiop'
+  /verifier/api/v1/authenticationresponse:
```

Comment:
Again, not rest. Since this basically updates the session, I think it could nicely be modeled as a ```PUT /verifier/api/v1/session/{state}```. It puts the credential into the session.